config.m4

```diff
@@ -82,12 +82,18 @@ AC_DEFUN([PHP_BSON_CLOCK],
 ])
 
 if test "$PHP_MONGODB" != "no"; then
-  AC_MSG_CHECKING([Check for supported PHP versions])
-  PHP_MONGODB_FOUND_VERSION=`${PHP_CONFIG} --version`
-  PHP_MONGODB_FOUND_VERNUM=`echo "${PHP_MONGODB_FOUND_VERSION}" | $AWK 'BEGIN { FS = "."; } { printf "%d", ([$]1 * 100 + [$]2) * 100 + [$]3;}'`
-  AC_MSG_RESULT($PHP_MONGODB_FOUND_VERSION)
-  if test "$PHP_MONGODB_FOUND_VERNUM" -lt "50500"; then
-    AC_MSG_ERROR([not supported. Need a PHP version >= 5.5.0 (found $PHP_MONGODB_FOUND_VERSION)])
+  AC_MSG_CHECKING([for PHP_CONFIG binary])
+  AC_CHECK_PROG(PHP_CONFIG_BINARY, [${PHP_CONFIG}], [yes], [no])
+  AC_MSG_CHECKING([supported PHP versions])
+  if test x"$PHP_CONFIG_BINARY" == x"yes"; then
```

Comment:
The PHP binary doesn't always exist... so we need to check for it.

Reply:
When was `PHP_CONFIG` introduced? Just curious where you found it missing.

Note that the leading "x" for equality checks is only necessary if an operand is potentially an empty string. In this case, `AC_CHECK_PROG()` will either assign "yes" or "no", so you're save comparing the variable to "yes" directly.